javatests/arcs/core/storage/StorageKeyFactoryTest.kt

```diff
@@ -0,0 +1,36 @@
+package arcs.core.storage
+
+import arcs.core.common.ArcId
+import arcs.core.data.Schema
+import arcs.core.data.SchemaFields
+import arcs.core.data.SchemaName
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+/** Tests for [StorageKeyFactory]. */
+@RunWith(JUnit4::class)
+class StorageKeyFactoryTest {
+
+    @Test
+    fun backingStore_namelessEntity() {
+        val emptySchema = Schema.EMPTY
```

Comment:
nit Schema.EMPTY is initialized with `setOf()` for names, but maybe worth asserting on that here, or adding an explicit schema with empty set of names (for the sake of the case EMPTY changes).

Reply:
👍  will decouple EMPTY from this test.